src/pages/OrderFinish/OrderFinish.tsx

```diff
@@ -0,0 +1,173 @@
+import { useState } from 'react';
+import CloseIcon from '@/assets/Main/close-icon.svg';
+import CallIcon from '@/assets/OrderFinish/call-icon.svg';
+import Motorcycle from '@/assets/OrderFinish/motorcycle-icon.svg';
+import Receipt from '@/assets/OrderFinish/receipt-icon.svg';
+import ShoppingCart from '@/assets/OrderFinish/shopping-cart-icon.svg';
+import Skillet from '@/assets/OrderFinish/skillet-icon.svg';
+import ArrowGo from '@/assets/Payment/arrow-go-icon.svg';
+import Button from '@/components/UI/Button';
+import Modal, { ModalContent } from '@/components/UI/Modal';
+import BottomModal, {
+  BottomModalHeader,
+  BottomModalContent,
+  BottomModalFooter,
+} from '@/components/UI/Modal/BottomModal';
+
+export default function OrderFinish() {
+  type OrderKind = 'order' | 'preparation' | 'delivery';
+
+  const [orderKind, setOrderKind] = useState<OrderKind>('order');
+
+  const [isModalOpen, setIsModalOpen] = useState<boolean>(false);
+  const [isBottomModalOpen, setIsBottomModalOpen] = useState<boolean>(false);
+  const [isCallBottomModalOpen, setIsCallBottomModalOpen] = useState<boolean>(false);
+
+  //const handleOpenModal = () => setIsModalOpen(true);
+  const handleCloseModal = () => setIsModalOpen(false);
+
+  const handleOpenBottomModal = () => setIsBottomModalOpen(true);
+  const handleCloseBottomModal = () => setIsBottomModalOpen(false);
+
+  const handleOpenCallBottomModal = () => {
+    setIsBottomModalOpen(false);
+    setIsCallBottomModalOpen(true);
+    setOrderKind('order'); //lint 에러 떠서 그냥 추가해놓음
+  };
+  const handleCloseCallBottomModal = () => setIsCallBottomModalOpen(false);
+
+  return (
+    <div className="flex flex-col">
+      <div className="flex flex-row justify-between px-6 py-4">
+        <div className="text-primary-500 flex h-[3.188rem] flex-col justify-center text-xl leading-[160%] font-bold">
+          주문 확인중
+          <div className="text-xs leading-[160%] font-normal text-neutral-500">사장님이 주문을 확인하고 있어요!</div>
+        </div>
+        <Button
+          onClick={handleOpenBottomModal}
+          className="h-[1.938rem] w-[4.125rem] self-end rounded-3xl px-3 text-xs leading-[160%]"
+        >
+          취소하기
+        </Button>
+      </div>
+      <div>
+        <div className="flex flex-row justify-between px-6 pt-[16px] pb-[6px]">
```

Comment:
```suggestion
        <div className="flex flex-row justify-between px-6 pt-4 pb-1.5">
```
tailwind에서 기본적으로 정의해주는 값들은 직접 적지 않고 이용하면 좋을 것 같아요